### ISSUE TITLE
Can I let it run with a custom domain available to the internet?

### ISSUE THREAD

**Issue by maxiwee69** [IBM/MAX-Image-Super-Resolution-Generator#67]
So I want to install it on my headless server but be able to use it on my phone via the browser but I don’t think there compatibility implemented here

**Comment by jleaders** [comment]
Yes any docker container can have it's own domain. This is outside the scope of this project to discuss, but chatGPT can step you through it.
Consider getting a namecheap name, using dyndns to your public IP, reverse proxy to 5000 with caddy2, and opening your port via your router. This is inherently insecure however, and proper security would be to isolate your docker host, your subnet, etc. etc. to prevent from a library that has a CVE
